cmd/classroom/service/get_room.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
```

Comment:
看到你大量地方用了这个包，这个包 21 年archived 了，原文如下
`Because of the Go2 errors changes, this package is not accepting proposals for new functionality.`

你可以尝试选择一个更好的包

Reply:
我现在全替换成fmt.Error("err: %w",err)的格式，这个官方库可以通过这个方法保留住原来的error，形成调用链